setup.py

```diff
@@ -119,7 +119,7 @@ def input_yes_no(self, key, prompt, default=u''):
         configured = self.data.get(key, default)
         prompt = u'\n{} Y/N [{}]: '.format(prompt, configured)
         while True:
-            inputed = raw_input(PROMPT + prompt + RESET) or configured
+            inputed = unicode(raw_input(PROMPT + prompt + RESET)) or configured
```

Comment:
`raw_input` vrati obycajny string a vsade chceme pracovat s unicode. Ak by tam nebol, do subor settings/configured.py by zapisal tiez len obycajny string. Je to spravne? Treba teraz doplnit `unicode` pri vsetkych `raw_input`?

Reply:
Ak `raw_input` vracia obycajny string, tak asi ano. Vsade chceme unicode stringy. Kludne to oprav (a pretestuj). Ale v samostatnom PR, kedze tu mocky zatian neriesime. 


Thread oznacujem ako resolved, kedze to spravime v samostatnom PR.